applications/DEMApplication/python_scripts/dem_default_input_parameters.py

```diff
@@ -43,6 +42,13 @@ def GetDefaultInputParameters():
             "strategy_parameters" :{
                 "RemoveBallsInitiallyTouchingWalls": false
             },
+            "problem_data"     : {
+                "problem_name"  : "dummy_name.Provide_a_real_one",
+                "parallel_type" : "OpenMP",
+                "echo_level"    : 1,
+                "start_time"    : 0.0,
+                "end_time"      : 1
+            },
```

Comment:
I think this is not used in the code, is it?

Reply:
It is, by que new approach (it calls de parent init) I added it to make this work.

Reply:
The json file will have a strange aspect with a "final_time" and an "end_time". Are you sure we need to add this part?

Reply:
@maceligueta This is not in the json that the user sees, only in the defaults.